Gemfile

```diff
@@ -185,3 +185,5 @@ group :development do
 
   gem 'rack-mini-profiler', '< 3.0.0'
 end
+
+gem "http", "~> 5.1"
```

Comment:
I can't remember! But it did the necessary job.

That's a great reference. Faraday is a clear winner in terms of popularity, and another webpage also shows the highest issue closure rate. It looks like it is super flexible and might be a good choice for HTTP operations in the future. But apparently we haven't needed it thus far. 

I can see in git history we once had HTTParty, which also looks like a solid option. But apparently we don't need it anymore.

So.. 🤷 this works, but I think Faraday would be a safer option, as it should be able to cater for our future needs and we can just learn the one API. It doesn't look like it would take much re-implementation. I'll have a look at the remaining comments before offering an opinion on if we should change or not.